### PR TITLE
Use brew instead of dpkg on OS X.

### PR DESCRIPTION
**Pull request by dragos**
Also, removed spurius \ in \cp.

### CODE REVIEW

File: mesos-docker/run/run.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ fi
 
 function get_latest_spark_version {
   echo "$(wget -qO- $MIRROR_SITE/mirror/apache/dist/spark/ | \
-  grep -oP "spark-[0-9].[0-9].[0-9]" | uniq | sort | tail -n1 | sed 's/spark-//g')"
```
Comment (skonto) [inline mesos-docker/run/run.sh]: I tested it on linux its fine without P

**Comment by skonto** [comment]
Are you going to resolve ./run/run.sh: line 166: apt-cache: command not found or do you want me to fix it?  LGTM.

**Comment by dragos** [comment]
Cool, thanks. I don't know how to fix `apt-cache` :)

**Comment by skonto** [comment]
:) shorted path, just an escape char: https://github.com/typesafehub/mesos-spark-integration-tests/pull/54

**Pull request merged**